WoloxAndroidBootstrp/app/src/main/java/ar/com/wolox/android/example/ui/viewpager/fragment/ViewPagerFragment.kt

```diff
@@ -58,9 +55,9 @@ class ViewPagerFragment private constructor() : WolmoFragment<FragmentViewpagerB
                 .show()
         }
 
-        logger.e("user uid", "${userSession.uid}")
-        logger.e("user Access-Token", "${userSession.accessToken}")
-        logger.e("isOnGoing", "${ userSession.isOngoingSession}")
+        Log.e("user uid", "${userSession.uid}")
+        Log.e("user Access-Token", "${userSession.accessToken}")
+        Log.e("isOnGoing", "${ userSession.isOngoingSession}")
```

Comment:
It is the correct way to log for development purposes. But don't forget to remove them after use. They cannot be in the code to prod.